server/handlers/converters_test.go

```diff
@@ -50,12 +49,10 @@ func TestMsgToPost(t *testing.T) {
 			},
 			setupPlugin: func(p *mocksPlugin.PluginIface, mockAPI *plugintest.API, client *mocksClient.Client, mockmetrics *mocksMetrics.Metrics) {
 				p.On("GetBotUserID").Return(testutils.GetSenderID())
-				p.On("GetURL").Return("https://example.com/")
 				p.On("GetClientForApp").Return(client)
 				p.On("GetMetrics").Return(mockmetrics).Maybe()
 			},
 			setupAPI: func(api *plugintest.API) {
-				api.On("LogDebug", "Unable to get user avatar", "Error", mock.Anything).Once()
```

Comment:
Seems like this can be simplified further.  We range through a slice of test cases that only has one test, and the setAPI call does nothing.  Should we just remove that?

Reply:
Sigh, you're right -- this test is basically doing nothing. I've got a separate ticket to revisit all of our tests as e2e / integration variety, and this one will probably just disappear in the process.